src/stactools/nisar_sim/stac.py

```diff
@@ -0,0 +1,149 @@
+
+    Returns:
+        Collection: STAC Collection object
+    """
+    summary_dict = {
+        "platform": c.NISAR_SIM_PLATFORMS,
+    }
+
+    collection = Collection(
+        id="nisar-sim",
+        title="Simulated NISAR",
+        description=c.NISAR_SIM_DESCRIPTION,
+        providers=c.NISAR_SIM_PROVIDERS,
+        extent=c.NISAR_SIM_EXTENT,
+        summaries=Summaries(summary_dict),
+        stac_extensions=[
+            ItemAssetsExtension.get_schema_uri(),
+            ProjectionExtension.get_schema_uri(),
+            SarExtension.get_schema_uri(),
+            # SatExtension.get_schema_uri(),
+        ],
+        keywords=c.NISAR_SIM_KEYWORDS,
+    )
+
+    # Links
+    collection.add_links(c.NISAR_SIM_LINKS)
+
+    # SAR Extension
+    sar = SarExtension.summaries(collection, add_if_missing=True)
+    sar.looks_range = c.NISAR_SIM_SAR["looks_range"]
+    sar.product_type = c.NISAR_SIM_SAR["product_type"]
+    sar.looks_azimuth = c.NISAR_SIM_SAR["looks_azimuth"]
+    sar.polarizations = c.NISAR_SIM_SAR["polarizations"]
+    sar.frequency_band = c.NISAR_SIM_SAR["frequency_band"]
+    sar.instrument_mode = c.NISAR_SIM_SAR["instrument_mode"]
+    sar.center_frequency = c.NISAR_SIM_SAR["center_frequency"]
+    # sar.resolution_range = c.NISAR_SIM_SAR["resolution_range"]
+    # sar.resolution_azimuth = c.NISAR_SIM_SAR["resolution_azimuth"]
+    # sar.pixel_spacing_range = c.NISAR_SIM_SAR["pixel_spacing_range"]
+    # sar.pixel_spacing_azimuth = c.NISAR_SIM_SAR["pixel_spacing_azimuth"]
+    # sar.looks_equivalent_number = c.NISAR_SIM_SAR["looks_equivalent_number"]
+
+    # sat = SatExtension.summaries(collection, add_if_missing=True)
+    # sat.absolute_orbit = c.NISAR_SIM_SAT["absolute_orbit"]
+
+    assets = ItemAssetsExtension.ext(collection, add_if_missing=True)
+    assets.item_assets = get_assets(collection=True)
+
+    return collection
+
+
+def create_item(product_path: str, dither: str) -> Item:
+    """Create a STAC Item
+
+    Args:
+        product_href (str): An HREF data directory pointing to a flight
+        dither (str): Type of dither ("X","G","D")
+            X: No Dither
+            G: Dither with gaps
+            D: Dithered without gaps
+
+    Returns:
+        Item: STAC Item object
+    """
+    dither = dither.upper()
+    if dither not in ["X", "G", "D"]:
+        raise ValueError("Dither value not valid")
+
+    metalinks = MetadataLinks(product_path, dither)
+    h5_data = HDF5Metadata(metalinks.h5_href)
+    ann_data = AnnotatedMetadata(metalinks.ann_href)
+    metadata = Metadata(product_path, metalinks.id, h5_data, ann_data)
+
+    item = Item(
+        id=metalinks.id,
+        properties={},
+        geometry=metadata.geometry,
+        bbox=metadata.bbox,
+        datetime=metadata.get_datetime,
+        stac_extensions=[],
+    )
+
+    proj_attrs = ProjectionExtension.ext(item, add_if_missing=True)
+    proj_attrs.epsg = 4326
+
+    # proj_attrs.shape = [1, 1]  # TODO: KMZ?
```

Comment:
What does KMZ have to do with these attributes?
Is this data gridded? if so it should have these attributes
https://github.com/stac-extensions/projection


property | type | description
-- | -- | --
proj:shape | [integer] | Number of pixels in Y and X directions for the default grid
proj:transform | [number] | The affine transformation coefficients for the default grid